src/Http/Client/Response.php

```diff
@@ -148,6 +133,20 @@ class Response extends Message implements ResponseInterface
         'headers' => '_getHeaders',
     ];
 
+    /**
+     * Map of deprecated magic properties.
+     *
+     * @var array
+     * @internal
+     */
+    protected $_deprecatedMagicProperties = [
+        'cookies' => 'getCookies()',
```

Comment:
For consistency, we could also not add () here, but add it when displaying it.
Then this could also be used more programmatically for example.